src/styles/parts/card.scss

```diff
@@ -0,0 +1,108 @@
+@import "../utils/_variables.scss";
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+  margin-right: 48px;
+  margin-bottom: 46px;
```

Comment:
You shouldn't use .card selector as it's BEM block. You need to add extra class which will be container's element like .cardContainer which includes .card and the html element with .card should have also .cardContainer__element class (name you as you wish, just follow container's name), only then BEM allows to set margin on BEM element (like .cardClass__element), on block (like .card) it's disallowed. It will work in the same way and it's more work to do but that's BEM rule.